src/tests/test_piranha_java.rs

```diff
@@ -59,13 +59,55 @@ create_rewrite_tests! {
   test_new_line_character_used_in_string_literal:  "new_line_character_used_in_string_literal",   1;
   test_insert_field_and_initializer:  "insert_field_and_initializer", 1;
   test_consecutive_scope_level_rules: "consecutive_scope_level_rules",1;
+  test_user_option_delete_if_empty: "user_option_delete_if_empty", 1;
+  test_user_option_do_not_delete_if_empty : "user_option_do_not_delete_if_empty", 1, delete_file_if_empty=false;
 }
 
 create_match_tests! {
   JAVA,
   test_java_match_only: "structural_find", 20;
 }
 
+#[test]
+
```

Comment:
Remove the empty line? (Not even sure if this breaks the pragma/annotation above... and even if it doesn't I don't see why we have the space)